src/java.desktop/share/classes/javax/swing/table/TableCellEditor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 1999, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 1999, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 1997, 2021, Oracle and/or its affiliates. All rights reserved.
```
I missed that: you only update the 2nd year, or add the 2nd one if there's only one yet.

Reply:
Updated the PR.

Reply:
"Do we keep the Javadoc as it is, or do we update it to be similar to TreeCellEditor?" means updating the interface definition?

Reply:
I'm referring to [this comment](https://github.com/openjdk/jdk/pull/6608#discussion_r764849899) and waiting for @prrace's opinion.

Reply:
Ok, got it.